sphinx/texinputs/Makefile_t

```diff
@@ -15,38 +21,33 @@ FMT = pdf
 
 {% if latex_engine == 'platex' -%}
 # latexmkrc is read then overridden by latexmkjarc
-LATEX = latexmk -r latexmkjarc -pdfdvi -dvi- -ps-
-PDFLATEX = $(LATEX)
-
-all-dvi: $(ALLIMGS) $(ALLDVI)
-
-all-ps: $(ALLPS)
-
-all: all-dvi $(ALLPDF)
-
-all-pdf: all-dvi $(ALLPDF)
-{% else -%}
-# latexmkrc contains config for pdflatex, lualatex, xelatex
+LATEX = latexmk -r latexmkjarc -dvi
+PDFLATEX = latexmk -r latexmkjarc -pdfdvi -dvi- -ps-
+{% elif latex_engine == 'pdflatex' -%}
 LATEX = latexmk -dvi
-{% if latex_engine == 'pdflatex' -%}
-LATEXMKPDFOPT = -pdf
+PDFLATEX = latexmk -pdf -dvi- -ps-
 {% elif latex_engine == 'lualatex' -%}
-LATEXMKPDFOPT = -pdflua
 LATEX = latexmk -lualatex
+PDFLATEX = latexmk -pdflua -dvi- -ps-
 {% elif latex_engine == 'xelatex' -%}
-LATEXMKPDFOPT = -pdfxe
-{% else -%}
-{% endif -%}
-PDFLATEX = latexmk $(LATEXMKPDFOPT) -dvi- -ps-
-
+# latexmk  $xelatex variable configured in latexmkrc to generate xdv files only
+LATEX = latexmk -xelatex
+PDFLATEX = latexmk -pdfxe -dvi- -ps-
+{% endif %}
+{% if latex_engine != 'xelatex' %}
 all-dvi: $(ALLIMGS) $(ALLDVI)
 
 all-ps: $(ALLPS)
+{% endif -%}
+{% if latex_engine == 'platex' %}
+all: all-dvi $(ALLPDF)
 
+all-pdf: all-dvi $(ALLPDF)
+{% else %}
 all: $(ALLPDF)
 
 all-pdf: $(ALLPDF)
-{% endif %}
+{%- endif %}
 
 %.png %.gif %.jpg %.jpeg:
 	extractbb $@
```

Comment:
@shimizukawa I incorporated from your Makefile but I see a problem here. For Japanese documents dvipdfmx is able to integrate pdf images. This is I presume the reason original Makefile had a for loop including all pdf files (not very satisfying). Does pdf image inclusion with dvipdfmx not need extractbb ?
By the way, it seems dvipdfmx since TeXLive 2015 but not earlier does the extractbb calls itself. (the extractbb binary is listed under safe executables which don't need shell-escape). Thus to test the above, one needs older installation.